test/unit/cli_test.py

```diff
@@ -37,7 +37,7 @@
                     identifier="E-TEST-1",
                     message="this is an error",
                     messagePlaceholders=[
-                        Placeholder(placeholder="param", description="")
+                        Placeholder(placeholder="param", description="description")
```

Comment:
Was this error in the unit test the reason why the mistake was not seen earlier?

Reply:
I feel, this change to this test is actually not enough, because now there are other cases that are not properly tested